sherpa/ui/utils.py

```diff
@@ -332,19 +333,85 @@ def reduce_ufunc(func):
 
 
 class ModelWrapper(NoNewAttributesAfterInit):
+    """Wrap up a model class so we can create instances easily.
+
+    The created instances can be used in a "model language" to create
+    complex expressions. If mdl1 and mdl2 have been created by
+    ModelWrapper then a user can say mdl1.n1 + mdl2.n2 to create model
+    instances named n1 and n2 and then return the expression which
+    represents their sum. Note that the model component is stored in
+    the session object.
+
+    """
+
+    def __init__(self, session, modeltype, args=(), kwargs=None):
+        # This is an internal class so do not bother with
+        # sherpa.utils.err exceptions.
+        #
+        if not isinstance(session, Session):
+            raise ValueError("session is not a Session instance")
+
+        if not _is_subclass(modeltype, Model):
+            raise ValueError("modeltype is not a Model instance")
```

Comment:
Not sure if the following suggestion is actually better or not.
```suggestion
            raise ValueError(f"{modeltype} is not a Model instance")
```

Reply:
Same again, we could say

    f"modeltype={modeltype} is not a Model instance"

Reply:
I like this suggestion that combines both of our initial ideas, but again, this should, as say, should never be hit, so not worth the effort?

Reply:
I've applied this change, as well as correcting the message, as it should say "Model class" not "Model instance".